frontend/app/src/components/molecules/nav-bar/nav-bar.tsx

```diff
@@ -105,7 +106,7 @@ function HelpDropdown() {
   );
 }
 
-function AccountDropdown({ user }: MainNavProps) {
+function AccountDropdown({ user, tenant }: MainNavProps) {
```

Comment:
I've been preferring to use `useTenant()` inside the components instead of drilling as a prop